money-spring/src/test/scala/com/comcast/money/spring/TracedMethodInterceptorScalaSpec.scala

```diff
@@ -124,6 +127,20 @@ class TracedMethodInterceptorScalaSpec extends WordSpec with Matchers with Mocki
       noteCaptor.getValue.isSticky shouldBe true
     }
   }
+  "TracedMethodInterceptor" should {
+    "not intercept a method not annotated by " in {
```

Comment:
You're right, that test doesn't need to be under it's own section.  Honestly the only reason I added the test was to increase coverage of that one class since the removal of so much other code means that the single missed line makes it seem that coverage dropped considerably, but the code in question is defensive and should never be called as the joinpoint specifies that the method requires that annotation.

I could probably change the code to assume that the method result would never be `null`.